assets/brunch-config.js

```diff
@@ -43,29 +43,28 @@ exports.config = {
   },
 
   conventions: {
-    // This option sets where we should place non-css and non-js assets in.
-    // By default, we set this to '/web/static/assets'. Files in this directory
-    // will be copied to `paths.public`, which is 'priv/static' by default.
-    assets: /^(web\/static\/assets)/
+  // This option sets where we should place non-css and non-js assets in.
+ // By default, we set this to "/assets/static". Files in this directory
+  // will be copied to `paths.public`, which is "priv/static" by default.
+    assets: /^(static)/
   },
 
   // Phoenix paths configuration
   paths: {
     // Dependencies and current project directories to watch
     watched: [
-      'web/static',
-      'test/static'
+      'static', "css", "js", "vendor"
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)